doc/script/collect_tex.py

```diff
@@ -16,8 +16,12 @@
   from ravenframework.util import InputData
 except ModuleNotFoundError:
   frameworkPath = os.path.join(teal_path, '..', 'ravenframework')
+  frameworkAltPath = os.path.join(teal_path, 'raven', 'ravenframework')
+  print("paths to try", frameworkPath, frameworkAltPath)
```

Comment:
do you want to retain this print?

Reply:
I can remove it, but it will never be used if TEAL and RAVEN are pip installed.

Reply:
Removed.